boot/boot.js

```diff
@@ -336,7 +336,7 @@ Get the browser location.hash. We don't use location.hash because of the way tha
 */
 $tw.utils.getLocationHash = function() {
 	var href = window.location.href;
-	var idx = href.indexOf('#');
+	var idx = href.indexOf("#");
```

Comment:
There seem to be a few of these corrections to existing core code. These corrections definitely need to be made, but don't belong in this PR and should be submitted to "master" as a separate PR that I can merge first.